setup.py

```diff
@@ -44,15 +40,9 @@
     packages=[
         "sdc.crypto",
         "sdc.crypto.scripts",
-        "sdc.crypto.test",
     ],
-    package_data={
-        "sdc.crypto": [
-            "requirements.txt",
-        ],
-    },
     scripts=['sdc/crypto/scripts/generate_keys.py'],
-    install_requires=installRequirements,
+    install_requires=["jwcrypto", "cryptography", "PyYAML"],
```

Comment:
:+1: We could use the Pipfile parser to get these, but I think the separation is good here.

Reply:
Have you got a link to that?  I couldn't find a decent way to get the dependencies from the pipfile

Reply:
https://github.com/pypa/pipfile/blob/master/pipfile/api.py

That's the parser, but I don't think you should use it. The requirements of the package installation should be kept seperate IMO.